packages/components/src/Table/TableRow.tsx

```diff
@@ -22,13 +22,10 @@ export const TableRow = ({ children, row }: TableRowProps) => {
   const ref = useRef(null);
   const { interactive, state, ...ctx } = useTableContext();
 
-  const { variant, size } = row.props;
```

Comment:
Is this no longer possible?

Reply:
to be honest, I don't actually see the use case

Reply:
should I add this question comment also again? ;)

Reply:
the idea was to have a highlighted row, maybe. but we can also readd it later if necessary :)